src/mode/mode_command.ts

```diff
@@ -17,23 +35,28 @@ export default class CommandMode extends Mode {
 
     HandleKeyEvent(key : string) : void {
         this.keyHistory.push(key);
+        
+        const editor = vscode.window.activeTextEditor;
+        const currentPosition = editor.selection.active;
+        
+        //var commands = vscode.commands.getCommands();
+        //commands.then(a => console.log(a));
 
-        switch (key) {
-            case ':':
-                showCmdLine();
-                break;
-            case 'h':
-                vscode.commands.executeCommand("cursorLeft");
-                break;
-            case 'j':
-                vscode.commands.executeCommand("cursorDown");
-                break;
-            case 'k':
-                vscode.commands.executeCommand("cursorUp");
-                break;
-            case 'l':
-                vscode.commands.executeCommand("cursorRight");
-                break;
+        var keyHandled = false;
+        for (var window = 1; window <= 2 && window <= this.keyHistory.length && keyHandled === false; window++) {
+            var keys : string  = _.takeRight(this.keyHistory, window).join('').toString();
+            
+            console.log(keys);
+            if (this.keyHandler[key] !== undefined) {
+                // Bug: 'two characters does not work'

```

Comment:
There's a bug right now where for some reason, I can't build a key with more than one chars.